src/function/get-weather-info.js

```diff
@@ -0,0 +1,21 @@
+import * as dotenv from "dotenv";
```

Comment:
why file name is words with - and function name is pascal case?

Reply:
I mistakenly thought it was best practice.
I renamed file to the same as the function.